packages/api/tests/endpoints/providers/update-provider.js

```diff
@@ -100,21 +100,15 @@ test.serial('PUT updates an existing provider and returns it in listing', async
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
 
-  t.plan(2);
-  const stub = sinon.stub(Search.prototype, 'query').resolves({
-    results: [updatedProvider]
-  });
-
+  t.plan(1);
   const response = await request(app)
-    .get('/providers')
+    .get(`/providers/${t.context.testProvider.id}`)
```

Comment:
Since we are no longer getting a listing of providers, the description of this test should be changed.  Right now it is "PUT updates an existing provider and returns it in listing".

Reply:
Since the other test has been updated to avoid collision with this one, is there a reason to change this test from listing providers to GETing the provider used in the test? It seems like the above test, `PUT updates an existing provider`, already tests that doing a GET returns the correctly updated provider

Reply:
@markdboyd  It didn't feel like the purpose of this test was intended to be to show that list works, so much as show that 'update' works.  It felt more future proof/better eating our own dog food  to explicitly call the endpoint to get the specific provider we want, instead of getting the full listing and parsing that.      Happy to change this if there's strong feelings in that regard. 

I've updated the title for now (good catch @yjpa7145 )

Reply:
f9a988a

Reply:
f9a988a

Reply:
Also in chatting with @markdboyd, given agreement on the above point, the previous test in this file becomes redundant. 

266382f1b